src/web/components/global-nav/top-nav/project-account-switcher/_lists/lists.html

```diff
@@ -1,4 +1,4 @@
-<div class='hig__global-nav__top-nav__project-account-switcher__lists'>
+<div class='hig__global-nav__top-nav__project-account-switcher__lists'>  
```

Comment:
Only diff here is a space - unless something was missed in a checkin, this should be undone.